io.catenax.return_request/2.0.0/ReturnRequest.ttl

```diff
@@ -0,0 +1,245 @@
+   samm:dataType :IdentificationEntity .
+
+:ProductConditionsEntity a samm:Entity ;
+   samm:preferredName "product conditions entity"@en ;
+   samm:description "Entity encapsulating the conditions the product is in."@en ;
+   samm:properties ( [ samm:property :missingParts; samm:optional true ] [ samm:property :dismantled; samm:optional true ] [ samm:property :mechanicalDamage; samm:optional true ] [ samm:property :corroded; samm:optional true ] [ samm:property :discolored; samm:optional true ] :mileage ) .
+
+:TimePeriodEntity a samm:Entity ;
+   samm:preferredName "time period"@en ;
+   samm:description "Entity defining a range of dates to define a time period."@en ;
+   samm:properties ( :startDate :endDate ) .
+
+:IdentificationEntity a samm:Entity ;
+   samm:preferredName "identification entity"@en ;
+   samm:description "Entity encapsulating different unique identifier of the product (manufacturingPartID, OeNumber and/or serialNumber)"@en ;
+   samm:properties ( :manufacturingPartID :oeNumber :serialNumber ) .
+
+:missingParts a samm:Property ;
+   samm:preferredName "missing parts"@en ;
+   samm:description "Completeness of the product, e.g. missing parts are not acceptable."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:dismantled a samm:Property ;
+   samm:preferredName "dismantled"@en ;
+   samm:description "Information on whether the product has been dismantled."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mechanicalDamage a samm:Property ;
+   samm:preferredName "mechanical damage"@en ;
+   samm:description "Information on mechanical damage to the part, e.g. two broken ribs or part deformation indicate mechanical damage."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:corroded a samm:Property ;
+   samm:preferredName "corroded"@en ;
+   samm:description "Information on whether the product has corrosion."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:discolored a samm:Property ;
+   samm:preferredName "discolored"@en ;
+   samm:description "Information on whether the product has been discoloured."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage"@en ;
+   samm:description "Current mileage of the component"@en ;
+   samm:characteristic :NumericValueofMileage ;
+   samm:exampleValue 89000 .
+
+:startDate a samm:Property ;
+   samm:preferredName "start date"@en ;
+   samm:description "The first day of the period"@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2019-01-01"^^xsd:date .
+
+:endDate a samm:Property ;
+   samm:preferredName "end date"@en ;
+   samm:description "The last day of the period."@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2030-09-30"^^xsd:date .
+
+:manufacturingPartID a samm:Property ;
```

Comment:
use of serialPart (e.g manufacturerPartId)

Reply:
Is a simple change of naming sufficient? (manufacturingPartID->manufacturerPartId)